app/services/windows.ts

```diff
@@ -203,6 +203,20 @@ export class WindowsService extends StatefulService<IWindowsState> {
     // This prevents "snapping" behavior when navigating settings
     if (options.componentName !== this.state.child.componentName) options.center = true;
 
+    // Override `options.size` when what is passed in is bigger than the current display
+    if (options.size) {
+      const { screen } = require('electron');
+      const { width: screenWidth, height: screenHeight } = screen.getPrimaryDisplay().workAreaSize;
```

Comment:
Primary display actually isn't what you want here.  You want to get the display that this window will appear on.  SLOBS is often run on secondary displays.

Reply:
Fixed